core/src/main/java/org/hisp/dhis/android/core/category/CategoryOptionComboCategoryLinkStoreImpl.java

```diff
@@ -37,14 +69,51 @@ public long insert(@NonNull CategoryOptionComboCategoryLinkModel entity) {
         return executeInsert();
     }
 
+    @Override
+    public int delete() {
+        return databaseAdapter.delete(CategoryOptionComboCategoryLinkModel.TABLE);
+    }
+
+    @Override
+    public boolean delete(@NonNull CategoryOptionComboCategoryLinkModel element) {
```

Comment:
In all stores in this method that delete by uid returns a int type, please can you refactor it?

Reply:
I changed the return type to an int but this entity is a link table and it has two primary keys if I remove the object I will have to have two uids instead of one object.

Do you have that I use two uids instead of one object?